app/components/TokenSale/TokenSalePanel/TokenSalePanel.jsx

```diff
@@ -6,9 +6,12 @@ import TokenSaleSelection from './TokenSaleSelection/TokenSaleSelection'
 import TokenSaleConditions from './TokenSaleConditions/TokenSaleConditions'
 import DialogueBox from '../../DialogueBox'
 import Button from '../../Button'
+// import Tooltip from '../../components/Tooltip/Tooltip'
```

Comment:
very minor but please remove all commented code before merging

Reply:
Of course. Good catch, we should setup lint rules for commented code. @comountainclimber